### ISSUE TITLE
How to show thank you page or success message after form submission.

### ISSUE THREAD

**Issue by netihj** [jamiewilson/form-to-google-sheets#59]
Hi Guys, This code is work like a charm. thank you so much.
Only issue I am facing is when I submit the form it goes to script.google page.
Is it possible to redirect on thank you page after successful submission?
Thank again for the wonderful code.

**Comment by Varshithvhegde** [comment]
Yes you can i have used something like this [refer my repo](https://github.com/Varshithvhegde/Contact_Form_Using_Google_Sheets/) , For live demo : https://varshithvhegde.me/Contact_Form_Using_Google_Sheets/

**Comment by netihj** [comment]
Thanks bro, you are really a Pro! 
Now only issue I am facing is, I am not receiving any email notification from google sheet, even tho I have enabled both settings. (see screenshot)
![image](https://user-images.githubusercontent.com/91874769/191159263-aaf85536-cf83-49cf-905a-e2a8acd060b8.png)

**Comment by Varshithvhegde** [comment]
After some reasearch i found out that notification will appear only if you submit the form using another email id . But the problem here is you are using api which is visible to anone and anyone can upload details to sheet een without a google account. That's the reason why you were not able to receive notification. I will let you know if I got any solution . But if you want contact form urgrntly and want notification feature as well you can use [https://formspree.io/](https://formspree.io/) which is free and also reliable but data may not e secure.
